configure.ts

```diff
@@ -42,5 +42,6 @@ export async function configure(command: Configure) {
    */
   await codemods.updateRcFile((rcFile) => {
     rcFile.addProvider('@adonisjs/i18n/i18n_provider')
+    rcFile.addMetaFile('resources/lang/**/*.json', false)
```

Comment:
Should be both `.json` and `.yaml` :)

Reply:
Fixed in https://github.com/adonisjs/i18n/commit/23a191aa9bb7ea8efcd893a0678ac24befb270dc